tasks/collect-data/README.md

```diff
@@ -27,6 +27,10 @@ should not be present in the Release data section).
 | snapshot             | Namespaced name of the Snapshot                    | No       | -             |
 | subdirectory         | Subdirectory inside the workspace to be used.      | Yes      | -             |
 
+## Changes in 4.3.0
+  * Task creates a results dir in the workspace (inside the subdirectory if it exists). The path to this
```

Comment:
> (inside the subdirectory if it exists)

The subdirectory actually gets created in this task. But only if it's set (i.e. non-zero string). So I think it should say: "(inside the subdirectory if set)"

Reply:
Updated